.circleci/config.yml

```diff
@@ -19,21 +19,48 @@ jobs:
           venv/bin/pip install --upgrade -r requirements.txt
     - run:
         name: PyTest with linters
-        # Google's client libraries will check for GOOGLE_APPLICATION_CREDENTIALS
-        # and use a file in that location for credentials if present;
-        # See https://cloud.google.com/docs/authentication/production
-        environment:
-          GOOGLE_APPLICATION_CREDENTIALS: /tmp/gcp.json
-        command: |
-          echo "${GCLOUD_SERVICE_KEY:?}" > "$GOOGLE_APPLICATION_CREDENTIALS"
-          venv/bin/pytest --black --docstyle --flake8 --mypy-ignore-missing-imports -n4
+        command: PATH="venv/bin:$PATH" script/entrypoint
     - save_cache:
         paths:
         - venv/
         key: python-packages-v1-{{ .Branch }}-{{ checksum "requirements.txt" }}-{{ checksum "constraints.txt" }}
+  release:
```

Comment:
following the pattern from `gcp-ingestion` of testing not-in-docker on PRs, but still testing in docker on merge to master.